app/(profile)/create-profile/page.tsx

```diff
@@ -12,9 +13,16 @@ const CreateProfilePage = async () => {
   }
 
   return (
-    <div>
-      Create profile page
-      <br />
+    <div className={styles.main}>
+      <div className={styles.titleBox}>
+        <span>Create profile page</span>
+        <button className={styles.saveProfileBtn}>
```

Comment:
**issue:** Button should be separate component. Try to think of it as you would do Nerdux UI system